packages/health-check/src/discord.js

```diff
@@ -0,0 +1,50 @@
+const { Client, MessageMentions } = require("discord.js");
+
+const client = new Client();
+
+const { DISCORD_BOT_TOKEN, PORTAL_NAME = "PORTAL_NAME not defined" } = process.env;
+
+client.on("ready", () => {
+  sendMessageToHealthCheckChannel(`${PORTAL_NAME}: reporting for duty!`);
```

Comment:
This happens only once, on portal startup, right?

Reply:
No, this happens when health-check service starts.

Reply:
Right, my question was more along the lines of is this a permanently running service that we'd start once and have running for a long time or is it going to be a short-lived one that would start up, do the checks, and then exit until it's time for it to run again.